Addons/testAddons.cpp

```diff
@@ -30,21 +30,29 @@
 
 #include "testConstructSurrogate.hpp"
 #include "testLoadUnstructured.hpp"
+#include "testExoticQuadrature.hpp"
 
-int main(int argc, char const **argv){
+void debugTest() {
+    cout << "Debug Test (callable from the CMake build folder)" << endl;
+    cout << "Put testing code here and call with ./Addons/addontester debug" << endl;
+    // debugGetRoots();
+    debugGetExoticQuadrature();
+    // debugSincT(100, 0.031353648322695503);
+    // debugSincT(10, 0.32099682841103033);
+    // debugSincT(1, 1.4321357541271255);
+}
 
-    cout << "\n\n";
```

Comment:
Make sure you don't delete this one.

Reply:
Noted. This line still remains in the latest commit.